app/frontend/Shared/FastList.svelte

```diff
@@ -57,6 +57,7 @@
   import { Collection, CollectionObserver, ArrayColl } from "svelte-collections"
   import { onMount, tick } from "svelte";
   import debounce from "lodash/debounce";
+  import { useThrottle } from "@svelteuidev/composables";
```

Comment:
This is a generic component, usable outside Mustang. It should not depend on svelteui.

Is the throttle necessary, given that your now scroll based on deltaY? For what exactly?